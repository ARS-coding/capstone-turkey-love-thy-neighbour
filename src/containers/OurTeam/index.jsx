import React from "react";

function OurTeam() {
  return <div />;
}

export default OurTeam;
