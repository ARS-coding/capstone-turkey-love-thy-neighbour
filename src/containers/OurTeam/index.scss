@import "../../style/variables";

  h2 {
    color: $logo-blue;
    text-align: center;
    padding-top: 80px;
    @media screen and (max-width: 768px){   
      padding-top: 50px;
      }
  }

  .ourteam-content-row {
    margin: 0 !important;
    padding-top: 80px;
    padding-bottom: 215px;
    display: flex;
    justify-content: space-around;
      @media screen and (max-width: 767px){   
        padding-bottom: 50px;
        padding-top: 40px;
        }

  svg {
    padding-right: 10px;
    height: 50%;
    display: flex;
    flex-direction: row;
     @media screen and (max-width: 374px){   
      width: 28px;
      height: 48px;
     }
     @media screen and (max-width: 991px) and (min-width: 768px) {    
      width: 28px;
      height: 48px;
     }
  }
  
 .img-fluid {
    @media screen and (max-width: 767px){      
      padding: 10px;              
    }
    @media screen and (min-width: 768px) {
      height: 100%;
      width: 100%;
      padding:5px
    }
 }

  .card {
    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
      rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
      height: 25%;
      width: 15%;
      padding-bottom: 5px;
      display: flex;
      justify-content: space-around;   
      @media screen and (max-width: 374px){   
        height: 90%;
        width: 100px;          
      }
      @media screen and (max-width: 424px) and (min-width: 375px) { 
        height: 90%;
        width: 160px;                           
      }
      @media screen and (max-width: 767px) and (min-width: 425px) {    
        height: 90%;
        width: 260px;         
      }
  }

   p{
    color: black;
    font-size: 90%;
      @media screen and (max-width: 1399px) and (min-width: 1200px){   
      font-size: 18px;
        }
      @media screen and (max-width: 1199px) and (min-width: 992px){   
      font-size: 15px;
        }
      @media screen and (max-width: 374px) {   
      font-size: 80%;
        }
  }

 .short-define{
      @media screen and (max-width: 991px) and (min-width: 768px){   
        margin-top: 20px;
        }
      @media screen and (max-width: 374px){   
        margin-top: 18px;
        }
  }

  .card-title {
    text-align: center;
    color: $logo-red;
    font-size: 16px;
      @media screen and (max-width: 1399px) and (min-width: 1200px){   
        font-size: 25px;
        }
      @media screen and (max-width: 1199px) and (min-width: 992px){   
        font-size: 20px;
        }
      @media screen and (max-width: 991px) and (min-width: 768px){   
        font-size: 14px;
        }
      @media screen and (max-width: 424px) and (min-width: 375px) {   
        font-size: 12px;
        }
      @media screen and (max-width: 374px) {   
        font-size: 80%;
      }
  }

  .card:hover {
    background-color: $logo-blue;
  }
}