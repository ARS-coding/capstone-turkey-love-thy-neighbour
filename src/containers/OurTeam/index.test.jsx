import React from "react";
import renderer from "react-test-renderer";
import OurTeam from ".";

describe("OurTeam Page", () =>
  it("renders OurTeam Page correctly", () => {
    const tree = renderer.create(<OurTeam />).toJSON();
    expect(tree).toMatchSnapshot();
  }));
