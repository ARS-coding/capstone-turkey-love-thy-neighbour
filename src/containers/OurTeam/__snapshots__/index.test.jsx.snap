// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurTeam Page renders OurTeam Page correctly 1`] = `
<div
  className="ourteam-content-container d-flex flex-column justify-content-center container"
>
  <h2>
    Here’s our hard working development team
  </h2>
  <div
    className="ourteam-content-row row"
  >
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="shrreya.jpeg"
      />
      <div
        className="card-title h5"
      >
         Shrreya Bhatachaarya 
      </div>
      <p
        className="short-define"
      >
        Lead Trainer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/Shrreya"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com/in/shrreya/"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com/__shrreya__"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="aliriza.jpg"
      />
      <div
        className="card-title h5"
      >
        Ali Rıza Şahin
      </div>
      <p>
        Junior Developer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/ARS-coding"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com/in/ars-coding/"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com/ARS_coding"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="kutay.jpg"
      />
      <div
        className="card-title h5"
      >
        Kutay Kağan Özen
      </div>
      <p>
        Junior Developer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/Iseluin"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="moulham.png"
      />
      <div
        className="card-title h5"
      >
        Moulham Hallak
      </div>
      <p>
        Junior Developer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/MoulhamHallak"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com/in/mahmoud-moulham-hallak-a2a688147/"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com/Moulham990"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="aziza.jpg"
      />
      <div
        className="card-title h5"
      >
        Aziza Iliasova 
      </div>
      <p>
        Junior Developer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/mimisha25"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com/in/mia-l-10b9a81b1/"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
    <div
      className="card d-flex flex-column align-items-center card"
    >
      <img
        alt="img"
        className="img-fluid"
        src="nur.jpg"
      />
      <div
        className="card-title h5"
      >
        Nur Hafise Bacaksiz 
      </div>
      <p>
        Junior Developer
      </p>
      <div
        className="social-media-logos d-flex flex-row justify-content-center col"
      >
         
        <a
          href="https://github.com/nurbacaksiz"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            github.svg
          </svg>
        </a>
         
        <a
          href="https://www.linkedin.com/in/h-nur-bacaks%C4%B1z-b352b2182"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            linkedIn.svg
          </svg>
        </a>
        <a
          href="https://twitter.com"
          rel="external noreferrer"
          target="_blank"
        >
          <svg>
            twitter.svg
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;
